Docker/docker-compose.yml

```
@@ -91,6 +93,14 @@ services:
       options:
         max-size: "200k"
         max-file: "10"
+  adaguc-redis:
+    image: redis:alpine
```

Shall we use the "defacto" image, and pin major version?
```suggestion
    image: redis:7
```